.NET/Microsoft.Recognizers.Text.Sequence/English/Parsers/PhoneNumberParser.cs

```diff
@@ -81,6 +83,12 @@ public double ScorePhoneNumber(string phoneNumberText)
             // Continue digit deduction
             score -= Math.Max(Regex.Matches(phoneNumberText, continueDigitRegex).Count - 1, 0) * continueDigitDeductionScore;
 
+            // Special award for special USphonenumber, i.e. 123-4567 or 123 - 4567
+            if (SpecialUSphonenumbeRegex.IsMatch(phoneNumberText))
+            {
+                score += specialUSphoneNumberAward;
```

Comment:
If I understand it right, this is to compensate some penalty from other scoring, right? If so, could it be relative to the relevant penalty, instead of a hardcoded value?

Reply:
sure, I'll